packages/plugins/src/foundry/decode.ts

```diff
@@ -280,98 +274,23 @@ export const makeParsedConfig = (
     }
 
     const { parsedContracts, unlabeledAdditionalContracts } =
-      parseAdditionalContracts(input, rawInputs, labels, configArtifacts)
+      parseAdditionalContracts(input, configArtifacts)
     unlabeledAddresses.push(...unlabeledAdditionalContracts)
 
     if (isRawCreate2ActionInput(input)) {
-      // Get the creation code of the CREATE2 deployment by removing the salt,
```

Comment:
The purpose of the following code was to infer the artifact based on the contract name, so none of it is necessary anymore